app/presenters/account_reset/pending_presenter.rb

```diff
@@ -0,0 +1,21 @@
+module AccountReset
+  class PendingPresenter
+    include ActionView::Helpers::DateHelper
+
+    attr_reader :account_reset_request
+
+    def initialize(account_reset_request)
+      @account_reset_request = account_reset_request
+    end
+
+    def time_remaining_until_granted
+      distance_of_time_in_words(
+        Time.zone.now,
+        account_reset_request.requested_at + Figaro.env.account_reset_wait_period_days.to_i.days,
+        true,
+        highest_measures: 2,
+        two_words_connector: I18n.t('datetime.dotiw.two_words_connector') + ' ',
```

Comment:
I noticed that the translations all have a leading space, and we're adding the trailing space here? WDYT of removing the space from the translations and then adding it here in front as well?

In my experience relying on formatting to remain the same across locales is fragile since that kind of stuff literally gets lost in translation and context across separate people

Reply:
We had to do some hacking to work w/ dotiw. By default it uses the array connectors from active support: https://github.com/rails/rails/blob/master/activesupport/lib/active_support/locale/en.yml#L32-L36

Those are expected to have spaces. Our YAML normalizer lops the spaces off of the end of lines. We threw this in here as a hack to work around that. I suppose the other approach would be to change the normalizer to not call `trim` on string values. We could do that?

Reply:
I think it's easier to manage if the YAML has no spaces at all and we add the spaces here:

```ruby
two_words_connector = [' ', I18n.t('....'), ' '].join
```